endpoints-support.md

```diff
@@ -73,6 +73,21 @@
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | PATCH    | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   |
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Interconnects**                                                                                                                            |
```

Comment:
Since we have removed API200/300 from the endpoints-support matrix due to space issue, shall we add a line here that these API's are still supported?

Reply:
We will be adding that in the  CHANGELOG with the changes we have made when we release this version

Reply:
ok..approved it.